src/dotnet/APIView/APIView/EventAPIV.cs

```diff
@@ -24,7 +24,9 @@ public EventAPIV(IEventSymbol symbol)
         {
             this.Name = symbol.Name;
             this.Accessibility = symbol.DeclaredAccessibility.ToString().ToLower();
-            this.Type = new TypeReference(symbol);
+            this.Type = new TypeReference(symbol.Type);
+            if (symbol.Type.SpecialType == SpecialType.System_String)
```

Comment:
Why is this outside of TypeReference ctor?

Reply:
I can't reference the SpecialType of an ISymbol directly, or at least haven't been able to. I'm sure there's some way to find it but I haven't been able to yet

Reply:
You can use `is` operator to check that type implement `ITypeSymbol`

`IsString = (symbol is ITypeSymbol typeSymbol) && typeSymbol.SpecialType == SpecialType.System_String`